azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go

```diff
@@ -298,14 +298,12 @@ func resourceArmKubernetesCluster() *schema.Resource {
 									"managed_outbound_ip_count": {
 										Type:          schema.TypeInt,
 										Optional:      true,
-										Computed:      true,
```

Comment:
Another option might be to allow this to be set to 0 and then in that case send nothing/0?